src/utils/style-formatter.js

```diff
@@ -110,12 +110,21 @@ export default {
       newFontsize *= button.clientWidth / text.offsetWidth;
     }
     // 4. Setting final font size by scaling with the font size from the layout + other font styling
-    if (hasIcon) {
+    if (hasIcon && style.font.style.italic) {
+      text.style.fontSize = `${Math.max(newFontsize * style.font.size * 0.84, 8)}px`;
+      text.children[0].style.marginRight = `${text.offsetWidth * 0.04}px`;
+      text.children[1].style.marginRight = `${text.offsetWidth * 0.04}px`;
```

Comment:
Still need this when overflow is visible?

Reply:
it makes large italic text look more centered, but to be fair if the text is very small then the offset is too much anyway
it's probably better without it I think